test/fixtures/flickr-search-app.html.out

```diff
@@ -36,7 +36,7 @@
         this.debounce('inputChanged', this.setOutput, this.delay);
       },
       setOutput: function () {
-        this.output = this.input;
+        this.set('output', this.input);
```

Comment:
Oh, very cool! I didn't realize that. Updated to only use `set` on complex paths.